packages/experiments/src/components/Chiclet/ChicletCard.base.tsx

```diff
@@ -49,13 +48,7 @@ export class ChicletCardBase extends React.Component<IChicletCardProps, {}> {
         {preview ? ( // render custom preview
           React.cloneElement(preview, { className: css(preview.props.className, customPreviewStyling) })
         ) : (
-          <Image
-            width={PREVIEW_IMAGE_WIDTH}
-            height={PREVIEW_IMAGE_HEIGHT}
-            src={image}
-            role="presentation"
-            alt={imageAlt ? imageAlt : undefined}
-          />
+          <img width={'auto'} height={'100%'} src={image} role="presentation" alt={imageAlt ? imageAlt : undefined} />
```

Comment:
If this `<img/>` has meaningful content in some or all cases, consider removing `role="presentation"`. See: https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/presentation/PresentationRoleExamples.html